### ISSUE TITLE
problem

### ISSUE THREAD

**Issue by monster-xie** [metehancekic/wireless-fingerprinting#2]
why use this complex nearual network model for ads-b signal get the result :

100/10239 [..............................] - ETA: 0s - loss: 2.6047 - acc: 0.2200
 3100/10239 [========>.....................] - ETA: 0s - loss: 2.7241 - acc: 0.1661
 5700/10239 [===============>..............] - ETA: 0s - loss: 2.7211 - acc: 0.1711
 8500/10239 [=======================>......] - ETA: 0s - loss: 2.7302 - acc: 0.1702
10239/10239 [==============================] - 0s 18us/step - loss: 2.7447 - acc: 0.1670

**Comment by metehancekic** [comment]
Hi, thanks for your interest. Can you provide which architecture that you use for the ads-b dataset? Because we use slightly different architectures for WiFi and ads-b. 

Details can be found on our paper (https://arxiv.org/pdf/2002.10791.pdf) on page 14 table (VIII).

**Comment by monster-xie** [comment]
> Hi, thanks for your interest. Can you provide which architecture that you use for the ads-b dataset? Because we use slightly different architectures for WiFi and ads-b.

the ads-b dataset is use usrp x310 to collet from real airplan，sample rate is 40M,class num is 38. There are >300 samples for each category. training set is  about 10000 samples,test is about 1000 samples.

**Comment by monster-xie** [comment]
@metehancekic  need some pre-process?

**Comment by metehancekic** [comment]
There is no need for preprocessing, however, the results provided for adsb on our paper sampled at 20m. Also we use different neural architectures for WiFi and ADS-B, details can be found on page 14 table VIII.

**Comment by monster-xie** [comment]
@metehancekic The network parameter is impress is very large ,now we modify the first layer kernel size =60 stride=60 filter=100
:
100/10239 [..............................] - ETA: 0s - loss: 0.5334 - acc: 0.8400
 3100/10239 [========>.....................] - ETA: 0s - loss: 0.5454 - acc: 0.8510
 5800/10239 [===============>..............] - ETA: 0s - loss: 0.5299 - acc: 0.8574
 8600/10239 [========================>.....] - ETA: 0s - loss: 0.5369 - acc: 0.8573
10239/10239 [==============================] - 0s 18us/step - loss: 0.5446 - acc: 0.8551

**Comment by monster-xie** [comment]
@metehancekic  Why the training acc 0.9,but the test acc is 0.36? dataset too small?

**Comment by metehancekic** [comment]
Are you using the preamble of the signals or whole signals? ADS-B signals can be 2 types: Mode S and Mode S extended, if you are using the whole signal, then it may cause performance issues in the test set when it locks onto ICAO address.

**Comment by monster-xie** [comment]
> Are you using the preamble of the signals or whole signals? ADS-B signals can be 2 types: Mode S and Mode S extended, if you are using the whole signal, then it may cause performance issues in the test set when it locks onto ICAO address.

I only using the preamble of signals. ADS-B signal is Modes extended.

**Comment by metehancekic** [comment]
It may be an overfitting issue, utilizing the dropout layer and/or weight decaying may improve your results.

**Comment by monster-xie** [comment]
> It may be an overfitting issue, utilizing the dropout layer and/or weight decaying may improve your results.
The complex raw data need some  normalized？

![image](https://user-images.githubusercontent.com/5821515/89750334-1cec0680-dafe-11ea-9a23-ece8a792a396.png)

**Comment by metehancekic** [comment]
We feed normalized signals in order to prevent the neural net to lock onto signal power. This image seems to have 320 samples, did you subsample it to have 20m sampling rate?

**Comment by monster-xie** [comment]
> We feed normalized signals in order to prevent the neural net to lock onto signal power. This image seems to have 320 samples, did you subsample it to have 20m sampling rate?

sampling rate is 40m,the image is only I channel of signals.

**Comment by metehancekic** [comment]
For ADS-B we found that phase information is very important, in other words, if we use only imaginary or only real part we got really low accuracies.

**Comment by monster-xie** [comment]
> For ADS-B we found that phase information is very important, in other words, if we use only imaginary or only real part we got really low accuracies.

Yes，before image is just example。could you provid part of ads-b signal dataset，I want to verify that there is a problem with the program.

**Comment by metehancekic** [comment]
Unfortunately, we are not allowed to share our dataset.

**Comment by monster-xie** [comment]
> Unfortunately, we are not allowed to share our dataset.

it doesn't matter，thank you for answer my questions！

**Comment by metehancekic** [comment]
No problem, you are welcome to ask questions.
